### ISSUE TITLE
"Seeking Help" challenge 2 answer & example misleading

### ISSUE THREAD

**Issue by delocalizer** [swcarpentry/r-novice-gapminder#256]
The explanation of the difference between `sep` and `collapse` is if not wrong, then pretty misleading - the actual difference between these is that `sep` string separates _terms,_ and `collapse` joins _results_ after concatenation. The example given only works because the arguments supplied are of length 1, which is a sort of degenerate case for `paste`. A clearer example would be: 
```
> paste(1:2, 3:4)
[1] "1 3" "2 4"
> paste(1:2, 3:4, sep = ',')
[1] "1,3" "2,4"
> paste(1:2, 3:4, sep = ',', collapse = '|')
[1] "1,3|2,4"
```
but this presumes a knowledge of vectors that the lesson hasn't yet reached at this point. So perhaps choose a different function than `paste` to use as an example?

**Comment by naupaka** [comment]
@delocalizer thanks for the comment. Which episode and line number is the current explanation on?

**Comment by delocalizer** [comment]
Hi @naupaka, it is episode 3, in challenge 2:
https://github.com/swcarpentry/r-novice-gapminder/blame/gh-pages/_episodes/03-seeking-help.md#L177

**Comment by naupaka** [comment]
@delocalizer I agree the current explanation is not so helpful. I also agree your example is clearer, but I think you're right about the problem about relying on a solid understanding of vectorized operations.  

What about:
```r
> paste(c("a","b"), "c")
[1] "a c" "b c"
> paste(c("a","b"), "c", sep = ",")
[1] "a,c" "b,c"
> paste(c("a","b"), "c", sep = ",", collapse = "|")
[1] "a,c|b,c"
```

**Comment by delocalizer** [comment]
Yes, that looks good to me

**Comment by naupaka** [comment]
@delocalizer want to submit a PR with this change?

**Comment by delocalizer** [comment]
sure thing


________________________________

**Comment by naupaka** [comment]
Closed by #262
